Sources/Clappr_iOS/Classes/Plugin/Core/MediaControl/MediaControlElement.swift

```diff
@@ -27,4 +27,10 @@ extension MediaControl {
             return .left
         }
     }
+    
```

Comment:
What do you think about moving the `enum` to the start of the escope, in the `extension` declaration, just to maintain the standard in code?

Reply:
Nice one. Was changed on: cf9d46f7ce2fe6afe627748935d03c385504882a